dev-util/librnp/librnp-0.17.1-r1.ebuild

```
@@ -28,6 +28,8 @@ DEPEND="${RDEPEND}
 BDEPEND="man? ( dev-ruby/asciidoctor )"
 
 src_configure() {
+	eapply ${FILESDIR}/0001-Removed-lookup-against-basic_string-uint8_t.patch
```

Please include at least the file name and preferably version too in the patch name, 
**librnp-0.17.1-Removed-lookup-against-basic_string-uint8_t.patch** or 
**librnp-0.17.1-libcxx-fix.patch**

also just add it to a `PATCHES` array so it gets processed correctly in `src_prepare()`. 

```suggestion
BDEPEND="man? ( dev-ruby/asciidoctor )"

PATCHES=( "${FILESDIR}"/librnp-0.17.1-Removed-lookup-against-basic_string-uint8_t.patch )

src_configure() {
```